### PR TITLE
Support port ranges 

### PR DESCRIPTION
**Pull request by jecnua**
Small typo :)

### CODE REVIEW

File: sync-pingdom-ec2-security-groups.py [hunk 1]
```diff
@@ -120,17 +120,22 @@ def main():
...
+        '--from-port',
         type=int,
         default=80,
-        help='The port on which Pingdom probes')
+        help='The port on which Pingdom probes (start of range)')
+    parser.add_argument(
+        '--to-port',
+        type=int,
+        default=443,
+        help='The port on which Pingdom probes (end of range)')
```
Comment (Stephan202) [inline sync-pingdom-ec2-security-groups.py]: This way the script defaults to opening all ports in the range [80, 443]; let's keep the default of only port 80.

File: README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ script with `--help`:
 $ ./sync-pingdom-ec2-security-groups.py --help
 usage: sync-pingdom-ec2-security-groups.py [-h] [--whitelist WHITELIST]
                                            [--protocol {icmp,tcp,udp}]
-                                           [--port PORT]
+                                           [--from-port PORT]
```
Comment (Stephan202) [inline README.md]: The other flag should be added as well.

**Review by Stephan202 — COMMENTED** [review]
This PR is no longer about a README fix; rather it is about a feature generalization. Which is cool, but let's stick with one feature per PR :). Since I realized the README is also outdated in some other respect, I moved that aspect to #5. Will merge that PR first, then get back to this one.

**Review by Stephan202 — APPROVED** [review]
I pushed some commits. If you're okay with the latest changes then I'll merge. Thanks for this feature!

**Comment by jecnua** [comment]
So sorry about that ^^' I thought I branched but I was working on master. Yes, I made some modifications to allow range after that, was not my plan to "pollute" the PR :)
Sorry again and if you like the changes please do merge all.
I will be more carefull next time ^^

**Comment by Stephan202** [comment]
No problem, eventually all came together nicely :)

Thanks for your contribution!

**Pull request merged**